interceptor/main.go

```diff
@@ -71,6 +71,18 @@ func main() {
 	logger.Infof("Server exiting")
 }
 
+func init() {
+	// read log level from environment variable
+	logLevel := os.Getenv("LOG_LEVEL")
+	if logLevel != "" || logLevel == "info" {
```

Comment:
Is it supposed to be `logLevel == "" || logLevel == "info"`? 

Either way I think what we want is two different loggers to load:
- one initialized on the cluster id
- one initialized without cluster id
Both of those should be able to receive a log level. Shall we move this logic into `logging.go`, and we'll be able to re-use accross interceptor, investigate and everything else?